src/cljs_time/core.cljs

```diff
@@ -187,6 +188,17 @@ expected."}
       (not= dayo dayother) (- dayo dayother)
       :else 0)))
 
+(defn get-week-year
```

Comment:
Could this go in `cljs-time.internal.core`? I'd rather not expose 2 new names doing the same thing.

Reply:
Can do.